docs/usage/shoot_high_availability_best_practices.md

```diff
@@ -269,13 +266,13 @@ As said, you can combine PTSCs with [pod affinities and/or anti-affinities](http
+Be aware however, that there are some limitations. Those are called [safeguards](https://kubernetes.io/docs/concepts/services-networking/topology-aware-hints/#safeguards) and if they strike, the hints are off and traffic is routed again randomly. Especially controversial is the balancing limitation as there is the assumption, that the load that hits an endpoint is determined by the allocatable CPUs in that topology zone, but that's not always, if even often, the case (see for instance [#113731](https://github.com/kubernetes/kubernetes/issues/113731) and [#110714](https://github.com/kubernetes/kubernetes/issues/110714)). So, this limitation hits far too often and your hints are off, but then again, it's about network latency and cost optimization first, so it's better than nothing.
 
 ## Networking
 
-We have talked about networking only to some small degree so far (`readiness` probes, pod disruption budgets, topology aware hints). The most important component is probably your ingress load balancer - everything else is managed by Kubernetes. AWS, Azure, GCP, and also OpenStack offer multi-zonal load balancers, so make use of them. In Azure and GCP, LBs are regional whereas in AWS and OpenStack, they need to be bound to a zone, which the cloud-controller-manager does by observing the zone labels at the nodes (please note that this behavior is not always working as expected, see [#570](https://github.com/kubernetes/cloud-provider-aws/issues/569) that we opened on the AWS cloud-controller-manager not readjusting to newly observed zones).
+We have talked about networking only to some small degree so far (`readiness` probes, pod disruption budgets, topology aware hints). The most important component is probably your ingress load balancer - everything else is managed by Kubernetes. AWS, Azure, GCP, and also OpenStack offer multi-zonal load balancers, so make use of them. In Azure and GCP, LBs are regional whereas in AWS and OpenStack, they need to be bound to a zone, which the cloud-controller-manager does by observing the zone labels at the nodes (please note that this behavior is not always working as expected, see [#570](https://github.com/kubernetes/cloud-provider-aws/issues/569) where the AWS cloud-controller-manager is not readjusting to newly observed zones).
 
-Please be reminded that even if you use a service mesh like [Istio](https://istio.io), the off-the-shelf installation/configuration usually never comes with productive qualities and you will have to fine-tune your installation/configuration, much like the rest of your workload.
+Please be reminded that even if you use a service mesh like [Istio](https://istio.io), the off-the-shelf installation/configuration usually never comes with productive settings (on purpose) and you will have to fine-tune your installation/configuration, much like the rest of your workload.
```

Comment:
Is the allegation that off-the-shelf installation/configuration does not come with productive settings 'on purpose' a bit strong and rather the opposite of defusing the wording?
I guess I understand what you mean, but it would probably be more fair to say that default installation/configuration often focuses on easy installation rather than implementing all details initially.

Reply:
LOL, with defusing I meant other changes. Here, I absolutely didn't want to accuse anyone of anything.

> I guess I understand what you mean, but it would probably be more fair to say that default installation/configuration often focuses on easy installation rather than implementing all details initially.

Yes, that obviously back-fired. Let me try it again. Thank you!